scripts/More_Options_to_Test/help.txt

```diff
@@ -70,6 +70,6 @@ Note: Use comments containing ' *INDENT-OFF*' and ' *INDENT-ON*' to disable
       processing of parts of the source file (these can be overridden with 
       enable_processing_cmt and disable_processing_cmt).
 
-There are currently 581 options and minimal documentation.
+There are currently 582 options and minimal documentation.
```

Comment:
Not related to this PR but generated files should never be under revision control.